sys/linux/socket_amd64.const

```diff
@@ -144,6 +144,7 @@ MSG_TRUNC = 32
 MSG_WAITALL = 256
 MSG_WAITFORONE = 65536
 MSG_ZEROCOPY = 67108864
+SCM_TXTIME = 61
```

Comment:
We also need updated for other arches. Did you try to run `make extract SOURCEDIR=/latest/kernel/sources/`? Or it does not work again?

Reply:
Run make extract SOURCEDIR=/latest/kernel/sources/ have no fail report. But it have some change in /sys/other_OS/gen/amd64.go after run make generate. Is it right?

Reply:
This is weird. gen/amd64.go should only change if *const files for that OS change. What is that other OS? And what does change in /sys/other_OS/gen/amd64.go?

Reply:
I get a lot of warnings "unsupported syscall: due to missing" when i run "make generate". Is it related?

Reply:
It is fine. Some syscalls are not supported on some arches.

Reply:
I have update the commit. There is change in /sys/other_OS/gen/amd64.go. Could you point out what's the wrong? Thanks!

Reply:
These changes are fine:
```
__BPF_FUNC_MAX_ID = 109
__NR_io_uring_enter = 426
```


Re changes in sys/akaros/gen/amd64.go it seems that you somehow does not have:
https://github.com/google/syzkaller/commit/162bd26be4e47d89f305435f7a1917b023138ed6
It did the changes that you PR now effectively undoes.

Reply:
syzkaller changes needs to be done based on syzkaller HEAD.
Do you have local changes to pkg/compiler that you did not check-in or something?

Reply:
It may caused by different version of clang-format. I seems is OK when i use clang-format from clang+llvm-6.0.1-x86_64-linux-gnu-ubuntu-14.04.

Reply:
Hard to believe that clang-format does semantic changes to our generated Go files...

Reply:
There is no change in /sys/other_OS/gen/amd64.go now. But Travis-CI build check is failed.

Reply:
Oh, please remove this block of code:
https://github.com/google/syzkaller/blob/4e7ecf0713aa5b07b0e2b9fffb6d6e51f759900d/prog/decl_test.go#L17-L22
This was a workaround for io_uring missing on arm/ppc. The test was especially made to fail when we have them so that we don't forget to remove that hack.

Reply:
Similarly, remove /prog/decl_test.go#line55-59 also?

Reply:
Yes